src/wdio/services/TerraCommands/chai-methods.js

```diff
@@ -28,8 +28,12 @@ function accessible() {
   *    (bool) and isExactSameImage (bool).
   * @property {bool} matchExactly - If the screenshots should be an exact match.
   */
-const getComparisonResults = (screenshots, matchExactly) => (
-  screenshots.map((comparison) => {
+const getComparisonResults = (screenshots, matchExactly) => {
+  if (screenshots.length < 1) {
+    return 'No screenshots to compare.';
```

Comment:
i'm not sure when this edge case occurs - i'm open to other ideas for the string we return.